Description: Added iwd sysvinit script for non-systemd users
 iwd (1.8-1+devuan3) UNRELEASED; urgency=medium
 .
   * Disabled compilation of systemd service.
   * Added a sysvinit script for non-systemd users.
Author: Job Bautista <jobbautista9@protonmail.com>
Last-Update: 2020-07-30

--- /dev/null
+++ iwd-1.8/iwd
@@ -0,0 +1,32 @@
+#!/bin/sh
+# Generated by sysd2v v0.3  --  http://www.trek.eu.org/devel/sysd2v
+# kFreeBSD do not accept scripts as interpreters, using #!/bin/sh and sourcing.
+if [ true != "$INIT_D_SCRIPT_SOURCED" ] ; then
+    set "$0" "$@"; INIT_D_SCRIPT_SOURCED=true . /lib/init/init-d-script
+fi
+### BEGIN INIT INFO
+# Provides:       iwd
+# Required-Start: $remote_fs
+# Required-Stop:  $remote_fs
+# Should-Start:   $network
+# Should-Stop:    $network
+# Default-Start:  2 3 4 5
+# Default-Stop:   0 1 6
+# Description:    Wireless service
+### END INIT INFO
+set -a
+STATE_DIRECTORY=/var/lib/iwd
+CONFIGURATION_DIRECTORY=/etc/iwd
+set +a
+
+DAEMON=/usr/libexec/iwd
+PIDFILE=/var/run/iwd-sysd2v.pid
+START_ARGS="--background --make-pidfile"
+
+do_start_cmd_override ()
+{
+  install -d -m 0700 /var/lib/iwd || return 2
+  install -d /etc/iwd || return 2
+  do_start_cmd || return $?
+}
+
